### PR TITLE
Update to hud.

### PR DESCRIPTION
**Pull request by calebkrauter**
Remove unnecessary details for controls with the @tclaar 's addition of controls screen.

Bug fix in weather system.

### CODE REVIEW

**Review by NathanHinthorne — APPROVED** [review]
Good changes Caleb!

File: js/classes/entities/foreground/weather_system.js [hunk 1]
```diff
@@ -24,17 +24,24 @@ class WeatherSystem {
...
             WeatherSystem.makePrecipitation();
         } else if (WeatherSystem.isClouds) {
+            WeatherSystem.clouds = true;
+            WeatherSystem.makeHeavens();
             WeatherSystem.makeClouds();
             // Default to sunny with white clouds
+        } else {
+            WeatherSystem.makeHeavens();
         }
-        WeatherSystem.makeHeavens();
```
Comment (tclaar) [inline js/classes/entities/foreground/weather_system.js:37]: Why'd you move this inside of the if-statement? You avoid duplicate code by keeping it here.

File: js/classes/entities/foreground/weather_system.js [hunk 2]
```diff
@@ -74,7 +81,7 @@ class WeatherSystem {
         // TODO add in dark clouds.
 
 
-        let cloudNum = cloudIntensity[WeatherSystem.intensity];
+        let cloudNum = cloudIntensity[2];
```
Comment (tclaar) [inline js/classes/entities/foreground/weather_system.js:84]: Does intensity no longer affect clouds?

File: js/classes/entities/midground/portal.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ class Portal {
     /**
      * @param {Vector} blockPos The position of the block the portal is on
      * @param {Number} type The type of portal (e.g. Portal.PURPLE or Portal.YELLOW)
+     * @author Nathan Hinthorne
```
Comment (tclaar) [inline js/classes/entities/midground/portal.js:5]: Why is there an author tag on this constructor?

File: js/classes/vector.js [hunk 4]
```diff
@@ -115,7 +115,7 @@ class Vector {
         return Vector.unit(Vector.subtract(b, a));
     };
 
-    /**
+    /**d
```
Comment (tclaar) [inline js/classes/vector.js:118]: Random d?

**Review by tclaar — APPROVED** [review]
Looks good!

File: js/classes/vector.js (hunk 4 above)
Comment (NathanHinthorne) [inline js/classes/vector.js:118]: I think you accidently left a "d" here in the JSDoc

**Pull request merged**